### PR TITLE
enh(README): rework GitHub project presentation

### PR DESCRIPTION
**Pull request by adr-mo**
- Rework the README
- Add CONTRIBUTING.md
- Add PULL_REQUEST_TEMPLATE.md
- Rework ISSUE_TEMPLATE.md
- Add CODE_OF_CONDUCT.md
- Add FEATURE_REQUEST_TEMPLATE.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -14,31 +21,36 @@ in the next 30 days, the ticket will be automaticaly closed.
...
+Please describe precisely the steps to reproduce the encountered issue.
 
+1. I logged in Centreon
+2. I reached the Custom View
+3. And so on...
 
-**Describe the results you expected:**
+<h3> Describe the received result </h3>
 
+<h3> Describe the expected result </h3>
```
Comment (jdelpierre) [inline .github/ISSUE_TEMPLATE.md]: this line is in duplicate

File: .github/FEATURE_REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Please describe your feature request in English.
+-->
+
+<h3> Feature explanation </h3>
+
+A short explanation of the wanted feature
+
+<h3> What is the value of the wanted feature ?</h3>
+
+A short explanation of the wanted feature.
```
Comment (ganoze) [inline .github/FEATURE_REQUEST.md]: Copy/paste of the previous chapter.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -14,31 +21,36 @@ in the next 30 days, the ticket will be automaticaly closed.
...
-**Centreon Web version**: 
+<h3> Prerequisites </h3>
 
-**Centreon Engine version**: 
+***Versions***
 
-**Centreon Broker version**: 
+-- Copy/Paste the result of the following command --
+```
+$ rpm -qa | grep centreon
```
Comment (ganoze) [inline .github/ISSUE_TEMPLATE.md]: This is only valid for RPM based systems.

We need components versions also on unpackaged systems (like Debian).

File: .github/FEATURE_REQUEST.md (hunk 2 above)
Comment (adr-mo) [reply .github/FEATURE_REQUEST.md]: Fixed.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (adr-mo) [reply .github/ISSUE_TEMPLATE.md]: Fixed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+<h2> Description </h2>
+
+Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
+* **QA Team** (Quality Assurance) with tests.
+* **reviewers** to understand what are the stakes of the pull request.
+
+Fixes # (issue)
+
+<h2> Type of change </h2>
```
Comment (ganoze) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is it also possible to add the target serie(s) to ease the cherry-picking ?

```
<h2> Target serie </h2>

Please delete series that are not relevant.

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x (master)
```

Comment (adr-mo) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes :+1:

Comment (adr-mo) [reply .github/PULL_REQUEST_TEMPLATE.md]: done :)
Thanks

**Pull request merged**